### DISCUSSION TITLE
Ffmpeg error catching.

### DISCUSSION THREAD

**Discussion by Russsgithub** [savonet/liquidsoap#1937] [Ideas]
Anybody know if I can (I haven't found any docs on this) or, if not, is it possible to add error catching for ffmpeg encoding.

In this case I'd like to be able to catch a audio incompatibility errors at an audio.copy ffmpeg encoder, and then change the encoder to something that will encode.

Thanks

**Comment by toots** [comment]
It's coming! See: https://github.com/savonet/liquidsoap/pull/1942

**Reply by Russsgithub** [reply]
Whoop, whoop. :)
